go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/kedacore/keda/v2
 
-go 1.22.1
-
-toolchain go1.23.3
```

Comment:
![image](https://github.com/user-attachments/assets/a62ac7f2-c1c9-49d2-b5f1-bba799807fd6)

When I add back the `toolchain` directive, `go mod tidy` removes it

Reply:
okay, let's keep it this way

Reply:
yeah, it's because the versions exactly match. But I suspect we won't always be dependent on the cutting-edge go, so the `toolchain` will come back :)